content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -54,6 +54,11 @@ Once you have a Linux-based Kubernetes control-plane node you are ready to choos
     sudo sysctl net.bridge.bridge-nf-call-iptables=1
     ```
 
+{{< note >}}
+The command should be run on all Linux nodes.
```

Comment:
instead of adding a separate note: `The following command must be run on all Linux nodes:` can replace the sentence on top `This can be done using the following command:`